ckan/lib/helpers.py

```diff
@@ -2344,6 +2344,35 @@ def mail_to(email_address, name):
     return html
 
 
+@core_helper
+def get_base_public_folder(app_conf):
```

Comment:
Or to be more clear, the `update_config` function is not called when doing `paster less`.